data/json/items/generic.json

```diff
@@ -3687,5 +3687,37 @@
     "cutting": 4,
     "to_hit": -3,
     "flags": [ "TRADER_AVOID", "NO_REPAIR" ]
+  },
+  {
+    "type": "TOOL",
+    "id": "training_dummy_light",
+    "symbol": "@",
+    "color": "brown",
+    "name": { "str": "training dummy" },
+    "category": "other",
+    "description": "A hand-made humanoid figure, useful for training in close quarters combat.  This one is made from wood and it has some duct tape to mark the striking points.",
+    "price": 15000,
+    "price_postapoc": 600,
+    "material": [ "wood" ],
+    "weight": "81500 g",
+    "volume": "62500 ml",
+    "bashing": 12,
+    "to_hit": -2
+  },
+  {
+    "type": "TOOL",
+    "id": "training_dummy_heavy",
```

Comment:
This doesn't have a recipe to make it?

Reply:
I want to use it for a higher tier practice recipe using "real weapons" (katanas, war hammers, halberds, etc.), but I haven't implemented it yet, I will add now the practice recipe and the recipe for the dummy, then I will mark the PR ready for review.